README.md

```diff
@@ -1 +1,19 @@
-# se701-assignment1
+# Threader
+
+![Build: Server](https://github.com/se701-group6/se701-assignment1/workflows/CD_SERVER/badge.svg)
+![Build: Frontend](https://github.com/se701-group6/se701-assignment1/workflows/CD_FRONTEND/badge.svg)
+
+Brought to you by SOFTENG 701 Group 6, Threader is a message board that lets users make text posts that other users
+can comment on.
+
+## Getting Started
+
+The application has two parts, the frontend and the server (or backend), which are in separate directories in this repository. Both parts need to be running for the application to work properly. To get everything set up, see:
+* [Backend tech page](https://github.com/se701-group6/se701-assignment1/wiki/Backend/)
+* [Frontend tech page](https://github.com/se701-group6/se701-assignment1/wiki/Frontend)
```

Comment:
This page doesn't exist yet, but I linked it under the assumption it will be created soon.

Reply:
I've created an empty page for it for now so the link works.